_locales/en/messages.json

```diff
@@ -74,9 +77,36 @@
   "editorFeatures": {
     "message": "Scratch Editor Features"
   },
+  "costumeEditorFeatures": {
+    "message": "Costume Editor Features"
+  },
+  "codeEditorFeatures": {
+    "message": "Code Editor Features"
+  },
   "websiteFeatures": {
     "message": "Scratch Website Features"
   },
+  "forumsFeatures": {
+    "message": "Forums Features"
+  },
+  "profilesFeatures": {
+    "message": "Profile Features"
+  },
+  "commentsFeatures": {
+    "message": "Comment Features"
+  },
+  "projectPlayerFeatures": {
+    "message": "Project Player Features"
+  },
+  "projectPageFeatures": {
+    "message": "Project Page Features"
+  },
+  "editorNavbarFeatures": {
+    "message": "Editor Navbar Features"
```

Comment:
I think "menu bar" is the correct word, although I'm not sure if we need a special category for addons that add options to the menu bar - especially because adding buttons to it isn't a very good idea (#2110).

Reply:
For now, I decided not to include this as a subcategory